sequencer/src/persistence.rs

```diff
@@ -451,18 +451,13 @@ mod persistence_tests {
             Some(vid_share3.clone())
         );
 
-        let mut seed = [0u8; 32];
-        let mut rng = rand_chacha::ChaChaRng::from_entropy();
-        rng.fill_bytes(&mut seed);
-
-        let tx = Transaction::random(&mut rng);
-        let tx_hash = Sha256::digest(tx.payload()).to_vec();
-        let block_payload_signature =
-            BLSPubKey::sign(&privkey, &tx_hash).expect("Failed to sign tx hash");
+        // TODO `leaf` is the genesis (empty) block; is that ok?
```

Comment:
Yes, this should be fine. The only thing that really matters for this test is the view number, since that's the key we use for GC

Reply:
comment removed in https://github.com/EspressoSystems/espresso-sequencer/pull/1607/commits/5afc03f0c9c8b3c95cc12d49d1513270bf194275